### ISSUE TITLE
Attached container configuration files

### ISSUE THREAD

**Issue by peregrinator** [vscode-kubernetes-tools/vscode-kubernetes-tools#1210]
In the docs, https://code.visualstudio.com/docs/devcontainers/attach-container#_attach-to-a-container-in-a-kubernetes-cluster, it's mentioned that "Attached container configuration files are not yet supported for containers in a Kubernetes cluster."

Is this already on the roadmap? Where might one look to begin working on this?
Thanks!

**Comment by zhuoqun-chen** [comment]
Yeah, I also want to ask about this feature, it seems that there will always be no configuration files for me and I don't know how to create one if it's already implemented. Sometimes my default always-should-be-installed extensions list in `settings.json` will be ignored and I need to manually install them when attaching to such running containers in my pod.

<img width="360" alt="image" src="https://github.com/user-attachments/assets/f95eea07-5504-4348-bcad-668590970087">

it may ensure the automation if it reads from a custom `devcontainer.json` file?

**Comment by msalbego93** [comment]
to change startup configuration in the way we attach to pods would be vital in the scenario where we have a read-only file system in pod and only a single mount volume (/tmp for example); so we could attach targeting to this specific folder that is writeable.
